# AudioCustomMsg.msg
std_msgs/Header header
uint8[] micLeft
uint8[] micRight
uint8[] micFront
uint8[] micRear
